Sources/LanguageServerProtocol/SupportTypes/ServerCapabilities.swift

```diff
@@ -773,21 +839,45 @@ public struct DocumentLinkOptions: WorkDoneProgressOptions, Codable, Hashable {
   }
 }
 
-public struct SemanticTokensOptions: WorkDoneProgressOptions, Codable, Hashable {
+public struct SemanticTokensOptions: WorkDoneProgressOptions, Codable, Hashable, LSPAnyCodable {
 
-  public struct SemanticTokensRangeOptions: Equatable, Hashable, Codable {
+  public struct SemanticTokensRangeOptions: Equatable, Hashable, Codable, LSPAnyCodable {
     public init() {
       // Empty in the LSP 3.16 spec.
     }
+
+    public init?(fromLSPDictionary dictionary: [String : LSPAny]) {
+      nil
```

Comment:
Shouldn’t this return an empty `SemanticTokensRangeOptions` so that encoding and decoding from `LSPAny` round-trips?

Reply:
hm... maybe you're right.
```swift
self = SemanticTokensRangeOptions()
```